plugins/TagFix_Deprecated.py

```diff
@@ -26,50 +26,62 @@
 
 
 class TagFix_Deprecated(Plugin):
-
-    def cleanWiki(self, src):
-        if src is None:
-            return src
-        src = src.replace("'''", "").replace("{{tag|", "").replace("{{Tag|", "").replace("}}", "").replace("<br/>", " ").replace("<br />", " ")
-        src = re.sub(r'([a-z])\|\|?([a-z*])', '\\1=\\2', src)
-        src = src.replace("|", "")
-        src = re.sub(r' +', ' ', src).strip()
-        return src
-
-
     def deprecated_list(self):
-        data = urlread(u"https://wiki.openstreetmap.org/wiki/Template:Deprecated_features?action=raw&force_cache_20180805", 1)
-        #data = open("Deprecated_features?action=raw").read()
-        data = data.split("{{Deprecated features/item")
-        dkey = re.compile(r"^\s*\|\s*dkey\s*=")
-        dvalue = re.compile(r"\s*dvalue\s*=")
-        suggestion = re.compile(r"^\s*\|\s*suggestion\s*=")
-        dataMult = []
-        for feature in data[1:]:
-            deprecated_key = None
-            deprecated_value = None
-            deprecated_suggestion = None
-            for line in feature.split("\n"):
-                if dkey.match(line):
-                    deprecated_key = line.split("|")[1].split("=")[1].strip()
-                    t = line.split("|")
-                    if len(t) > 2:
-                        if dvalue.match(t[2]):
-                            deprecated_value = t[2].split("=")[1].strip()
-
-                if suggestion.match(line):
-                    deprecated_suggestion = line.split("=")[1].strip()
-
-                dataMult.append((deprecated_key, deprecated_value, deprecated_suggestion))
+        wikiRoot = 'https://wiki.openstreetmap.org/wiki'
+        data = urlread(wikiRoot + '/Template:Deprecated_features?action=raw', 1)
+
+        # Tidy data up for processing
+        # Eliminate wiki bold formatting
+        data = data.replace("'''", "")
+
+        # Remove HTML newlines
+        data = re.sub(r'<br\s*/>', ' ', data)
+
+        # Remove excess whitespace (also removes all newlines)
+        data = " ".join(data.split())
+
+        # Eliminate any whitespace around pipe characters
+        # This makes reading the template parameters simple
+        data = re.sub(r'\s?\|\s?', '|', data)
+
+        # Eliminate templates to prevent unexpected pipe characters
+        data = re.sub(r'{{{\s?lang\s?\|?\s?}}}', '', data, flags=re.I)
+        # Tag template can take one or two params, with trailing | possible
+        data = re.sub(
+            r'{{Tag\s?\|(.+?)\|?\s?}}',
+            lambda x : '`{}`'.format(x[1].replace("|", "=")),
```

Comment:
Got this here
```
2020-02-10 17:26:19         lambda x : '`{}`'.format(x[1].replace("|", "=")),
2020-02-10 17:26:19     TypeError: '_sre.SRE_Match' object is not subscriptable
```

Reply:
Ah apparently the `__getitem__` method was only added to match objects in 3.6, will update